server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
yes it was kind of added on purpose.. it was becuase i needed to run https for dropboxauth to work... and was trying to get livereload to work over https also.

its just a dummy selfsigned key so not secret or anything